project-base/storefront/cypress/tsconfig.json

```diff
@@ -11,5 +11,5 @@
         "incremental": true,
         "esModuleInterop": true
     },
-    "include": ["**/*.ts", "**/*.cy.ts", "../graphql/generated/index.tsx"]
+    "include": ["**/*.ts", "**/*.cy.ts", "../**/*.ts", "../**/*.tsx"]
```

Comment:
"include": ["**/*.ts", "../**/*.tsx"]

Reply:
I changed it to:

`"include": ["**/*.ts", "../**/*.tsx", "../**/*.ts"]`

we need `"../**/*.ts"` to also include SF files (we need to go one level up)

Reply:
Ah, yes, I've missed that.